src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -122,9 +115,12 @@ public double getY(){
   private GenericEntry elbowOutput = armTab.add("Elbow Output", 0.0).getEntry();
   private GenericEntry armXPosition = armTab.add("Arm X Position", 0.0).getEntry();
   private GenericEntry armYPosition = armTab.add("Arm Y Position", 0.0).getEntry();
+  private GenericEntry elbowP = armTab.add("Elbow P", kElbowP).getEntry();
+  private GenericEntry elbowI = armTab.add("Elbow I", kElbowI).getEntry();
+  private GenericEntry elbowD = armTab.add("Elbow D", kElbowD).getEntry();
 
-  private final SingleJointedArmSim baseArmSim = new SingleJointedArmSim(DCMotor.getNEO(1), kBaseGearing, baseArmInertia, kBaseArmLength, kMinBAngle, kMaxBAngle, kBaseArmMass, false);
-  private final SingleJointedArmSim elbowArmSim = new SingleJointedArmSim(DCMotor.getNEO(1), kElbowGearing, elbowArmInertia, kElbowArmLength, kMinEAngle, kMaxEAngle, kElbowArmMass, false);
+  private final SingleJointedArmSim baseArmSim = new SingleJointedArmSim(DCMotor.getNEO(2), kBaseGearing, baseArmInertia, kBaseArmLength, kMinBAngle, kMaxBAngle, false);
+  private final SingleJointedArmSim elbowArmSim = new SingleJointedArmSim(DCMotor.getNEO(1), kElbowGearing, elbowArmInertia, kElbowArmLength, kMinEAngle, kMaxEAngle, false);
```

Comment:
Why remove the masses?

Reply:
The constructor for an arm sim doesn't take them anymore, I think it was an update. It just uses the MOI now